mpop_kiosk/PageSingleQuestion.qml

```diff
@@ -13,9 +13,12 @@ ColumnLayout {
     property var datavizSender: null
     property string minText: ""
     property string maxText: ""
-    property string titleText: ""
+    property string titleText: "Question 01"
```

Comment:
This should be provided where we instanciate the widget, not in the "class" file of the widget.